### ISSUE TITLE
debounce(componentWillReceiveProps) breaks with MobX + multiple instances

### ISSUE THREAD

**Issue by ericclemmons** [JedWatson/react-codemirror#63]
I just confirmed that when when an `observable` changes (e.g. `@observable myName = ""`):

``` js
<CodeMirror key="something" value={myName} />
<CodeMirror key="something-else" value={myName} />
```

Only the **last** `<CodeMirror />` instance updates.

Once I removed `debounce` from `componentWillReceiveProps`, everything worked as intended.

**Comment by afc163** [comment]
I use `{myName && <CodeMirror key="something" value={myName} />}` as workaround.

**Comment by JedWatson** [comment]
Should be fixed with v0.3.0, please reopen if not!
